.reuse/dep5

```diff
@@ -0,0 +1,44 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: talk-desktop
+Upstream-Contact: Nextcloud <info@nextcloud.com>
+Source: https://github.com/nextcloud/talk-desktop/
+
+# L10n
+Files: l10n/*.js l10n/*.json
+Copyright: 2022 Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+# CI
+Files: .github/**/*
+Copyright: none
+License: CC0-1.0
+
+# Common configs
+Files: package.json package-lock.json */package.json */package-lock.json composer.json composer.lock */composer.json */composer.lock .gitignore .l10nignore psalm.xml tests/psalm-baseline.xml vendor-bin/*/composer.json vendor-bin/*/composer.lock .tx/config */phpunit.xml tsconfig.json
+Copyright: none
+License: CC0-1.0
+
+# Other configs
+Files: .env.example .browserslistrc .editorconfig .eslintrc.js .gitattributes
+Copyright: none
+License: CC0-1.0
+
+# Vue Devtools
+Files: resources/vue-devtools.crx
+Copyright: Evan You <https://github.com/vuejs/devtools/blob/main/LICENSE>
+License: MIT
+
+# Sounds
+Files: sounds/notification.ogg sounds/talk.ogg
+Copyright: 2017 Feandesign https://soundcloud.com/feandesign
+License: CC0-1.0
+
+# Nextcloud marks
+Files: img/**/*
+Copyright: Nextcloud GmbH <https://nextcloud.com/trademarks/>
+License: LicenseRef-NextcloudTrademarks
+
+# Nextcloud server styles and assets, generated by the server
+Files: src/shared/assets/default/**/*
+Copyright: Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
```

Comment:
Not sure if this is correct. This is styles generated by the server + from the server source

Reply:
@nickvergessen Question about license is still actual here 👀